docs/data/joy/components/list/ExampleCollapsibleList.tsx

```diff
@@ -31,14 +31,12 @@ export default function ExampleCollapsibleList() {
             '--joy-palette-text-secondary': '#635e69',
             '--joy-palette-primary-plainColor': '#d48cff',
           },
-
```

Comment:
I think a saw a PR for fix this, but in case it was unrelated, we shouldn't transform these lines.